docs/add-ons/configuration.md

```diff
@@ -152,7 +152,7 @@ image: repo/{arch}-my-custom-addon
 | privileged | list | no | Privilege for access to hardware/system. Available access: `NET_ADMIN`, `SYS_ADMIN`, `SYS_RAWIO`, `SYS_TIME`, `SYS_NICE`, `SYS_RESOURCE`, `SYS_PTRACE`, `SYS_MODULE` or `DAC_READ_SEARCH`
 | full_access | bool | no | Give full access to hardware like the privileged mode in Docker. Works only for not protected add-ons. Consider using other add-on options instead of this, like `devices`. If you enable this option, don't add `devices`, `uart`, `usb` or `gpio` this is not needed.
 | apparmor | bool/string | no | Enable or disable AppArmor support. If it is enable, you can also use custom profiles with the name of the profile.
-| map | list | no | List of maps for additional Home Assistant folders. Possible values: `config`, `ssl`, `addons`, `backup`, `share` or `media`. Defaults to `ro`, which you can change by adding `:rw` to the end of the name.
+| map | list | no | List of maps for additional Home Assistant folders to mount within the add-on. Possible values: `config`, `ssl`, `addons`, `backup`, `share` or `media`. Defaults to `ro`, which you can change by adding `:rw` to the end of the name. These folders are relative to the Supervisor directory. On Home Assistant OS, the Supervisor directory is at [`/mnt/data/supervisor/`](https://www.home-assistant.io/common-tasks/os/#configuring-access-to-files).
```

Comment:
Right. I was proposing the documented Home Assistant OS directory as a point of reference. When you first approach add-ons, you experience containers upon containers, and it helps to have some ground truth on where in the filesystem these files exist.

Reply:
How does that help? As the file system is not regularly accessible (and one doesn't need to access it for add-on development purposes either).

Reply:
Sorry, maybe I am not clear on the subject, **why** would one need to know that? In general, one should not (ever) need to access the filesystem on that level. All these folders are available within the user space, using e.g., the ssh & terminal add-ons, samba add-on, FTP add-on...

There is no need to access the filesystem of the OS for any of this. Hence I try to ask time and time again. Why this is needed?

You seem to not be able to answer the question or are missing the point of my question.

Reply:
Sorry that this has caused confusion. As someone approaching add-on development for the first time, I know that I can map these directories using the `map` config value:

* addons
* backup
* config
* media
* share
* ssl

But, after spending time with the docs, it's not clear how to use each of these directories, and how they interact with other add-ons and Home Assistant itself. If there was a way to get more clarity on that, then that's what I'm after. (Absent such a doc, reviewing the filesystem was the only way I could make sense of it.) Is there an existing doc that I'm missing?

As for this PR, we can close it, because it's not helping to clarify toward that goal. Thanks for taking the time to review and ask questions. Let's close and move on.

Reply:
>  it's not clear how to use each of these directories,

> Is there an existing doc that I'm missing?

That is already to be found in the rest of the documentation.
It also visible in the introduction tutorial: <https://developers.home-assistant.io/docs/add-ons/tutorial>

Additionally, it is available thought-out all other documentation we have. As these are the folders in the eco system. The OS is not a concern for any of us on this level. It is abstracted away and one does not need to access it.

Reply:
Thanks